media/inf/syssetup.inf

```diff
@@ -144,3 +144,22 @@ Control="System\CurrentControlSet\Control"
 
 [Classes]
 HKLM,%CurrentVersion%\Installer,"InstallerLocation",,"%11%"
+
+[DestinationDirs]
+IECopy=16422,Internet Explorer
+
+[Finalize] ; Executed by RunOnceEx on first normal boot
+; FIXME: This should be moved to ie.inf,DefaultInstall and executed by [Infs.Always]
+AddReg=FinalizeAddReg
+CopyFiles=IECopy
+RegisterDlls=IEReg
+
+[FinalizeAddReg]
+HKLM,SOFTWARE\Microsoft\.NETFramework,InstallRoot,,"%10%\Microsoft.NET\Framework\"
```

Comment:
Question for my understanding: What has the .NET thingy to do with IE? From my understanding: nothing. So that FinalizeAddReg-part from my understanding should not be moved to ie.inf ultimately, but somewhere else instead. But **where**?

Reply:
It has nothing to do with IE but it also causes a ugly console window popup on first boot.

Reply:
Yes sure, that question wasn't raised to hold back merging your PR in any way or question what you did here. I just raised it so that we can later remember that it shouldn't be moved together with the other stuff, because the comment currently is placed directly before the FinalizeAddReg. Not sure which inf in Windows 2k3sp2 is responsible for .NET-things.

Reply:
We discussed this in Town Square last night and we can't think of a reason to delay this to first boot in the first place, I will move it to another part of syssetup.inf.

Reply:
good idea. Thumbs up for that plan.